### ISSUE TITLE
Knitting to make HTML

### ISSUE THREAD

**Issue by mchaji1** [cssearcy/AYS-R-Coding-SPR-2020#4]
Hello, 
This is my first time using the knit function in R and when finishing up I received the following: file:///Users/dawn/Desktop/Lab-01-Chaji.html
I am a little skeptical because the HTML looks just like a pathway on my computer. Is this typical? 

Thank you!

**Comment by jamisoncrawford** [comment]
@mchaji1 You're correct, that's the path to your file, stored locally on your machine.

Before you knit your `.Rmd` file, RStudio makes you save it to a local directory of your choosing.

Wherever you save the `.Rmd` file, once knit, you'll find the `.html` output in the same directory.

Hope this helps!

**Comment by mchaji1** [comment]
Thank you so much!
